### PR TITLE
Update to last rustc version

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @gkoz

### CODE REVIEW

File: src/cairo_threads.rs [hunk 1]
```diff
@@ -163,8 +164,8 @@ fn draw_initial(format: Format, width: i32, height: i32) -> (Box<[u8]>, i32) {
     (buf, image.get_stride())
 }
 
-fn draw_slow(cr: &Context, delay: u32, x: f64, y: f64, radius: f64) {
-    thread::sleep_ms(delay);
+fn draw_slow(cr: &Context, delay: u64, x: f64, y: f64, radius: f64) {
```
Comment (gkoz) [inline src/cairo_threads.rs]: I guess `delay` should be `Duration` then?

**Comment by gkoz** [comment]
I think we push to `pending` first and then could cherry-pick into the other branches if you'd like this to not wait for new crates.

**Comment by GuillaumeGomez** [comment]
As you prefer, I don't have preference. I just want to avoid disturbing your current work.

File: src/cairo_threads.rs (hunk 1 above)
Comment (GuillaumeGomez) [inline src/cairo_threads.rs]: I find it easier to understand this way. It's like a little wrapping.

Comment (gkoz) [inline src/cairo_threads.rs]: If the standard library "suggests" we take advantage of richer types i.e. adopt the more specific `Duration` type it seems logical to do this fully.

**Comment by gkoz** [comment]
This shouldn't matter and the commit could be cherry-picked in any direction. But generally I'd prefer consistency ;)

**Comment by gkoz** [comment]
I think it would be simpler to reopen this against the `pending` branch.

**Pull request closed without merging**